src/DynamoRevit/DynamoRevit.cs

```diff
@@ -607,7 +607,8 @@ private static RevitDynamoModel InitializeCoreModel(DynamoRevitCommandData comma
                 AuthProvider = new RevitOAuth2Provider(new DispatcherSynchronizationContext(Dispatcher.CurrentDispatcher)),
                 ExternalCommandData = commandData,
                 UpdateManager = revitUpdateManager,
-                ProcessMode = isAutomationMode ? TaskProcessMode.Synchronous : TaskProcessMode.Asynchronous
+                ProcessMode = isAutomationMode ? TaskProcessMode.Synchronous : TaskProcessMode.Asynchronous,
+                Preferences = PreferenceSettings.Instance
```

Comment:
Do we need to pass the PathResolver?

Reply:
> Do we need to pass the PathResolver?

yes as the previously been doing in the startconfiguration, now We assigning it to the PathManager in the DynamoModel ctor, the only update is need to pass the Preferences.